.github/workflows/release.yaml

```diff
@@ -0,0 +1,59 @@
+on:
+  release:
+    types: [published]
```

Comment:
Does this mean that this action is only triggered when a new github release is manually created? If yes, is there some way how to make it more like the travis flow, where a release was created automatically once a tagged commit was merged to master.

Reply:
That's a good question, I will look into it, I haven't considered releases being created from a tagged commit

Reply:
@akrejcir I think we should leave it as-is for now, until we have an automatic release automation, otherwise it makes no sense

Reply:
Ok. Then we will need to always remember to create a tag before release.

Reply:
No need, a tag is created automatically when you create a manual release on Github

Reply:
I didn't know that. Then there is no problem.